.github/workflows/ncc.yml

```
@@ -1,21 +1,14 @@
 name: ncc
 on:
   push:
-    tags-ignore:
+    # Can't push a build commit to a tag, so only run for branches
+    branches:
       - '**'
     paths:
+      # Include any files that could require rebuilding
       - 'package-lock.json'
       - 'src/**'
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 14
-      - run: npm ci
-      - uses: planningcenter/balto-utils/ncc@main
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+  ncc-build:
+    uses: planningcenter/balto-utils/.github/workflows/ncc.yml@v1
```

OPTIONAL if you'd prefer to manage setup yourself
```suggestion
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: actions/setup-node@v3
        with:
          node-version: 14
      - run: npm ci
      - uses: planningcenter/balto-utils/ncc@main
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
```